plugins/services/src/js/events/MarathonActions.js

```diff
@@ -242,6 +244,44 @@ var MarathonActions = {
     });
   },
 
+  resetDelayedService(service, force = false) {
+    if (!(service instanceof Service)) {
+      if (process.env.NODE_ENV !== "production") {
+        throw new TypeError("service is not an instance of Service");
+      }
+
+      return;
+    }
+
+    let url = buildURI(`/queue/${service.getId()}/delay`);
+
+    if (service instanceof Pod) {
+      url = buildURI(`/queue/${service.getId()}/delay`);
+    }
```

Comment:
Could we remove this conditional block? Seems to be the same as the original assignment.

Reply:
I will check if pods use the same function.

-

Reply:
You are right, we can remove it.

Reply:
Done.